src/api/fetchData.ts

```diff
@@ -0,0 +1,23 @@
+import { BASE_URL } from '../constants/constants';
+
+const posts = 'posts';
+const users = 'users';
+const comments = 'comments';
```

Comment:
тоже в верхнем регистре, правда не очень понятно, данные тут или строки для fetch
```suggestion
const API_POSTS_POSTFIX = 'posts';
const API_USERS_POSTFIX = 'users';
const API_COMMENTS_POSTFIX = 'comments';
```

что-то в таком духе, или придумать чтобы было понятно что это ключевые слова добавляемые к ссылкам